cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/BudgetDTO.java

```diff
@@ -5,6 +5,7 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.liberologico.cloudesire.common.MathConfiguration;
 import com.liberologico.cloudesire.common.MathUtils;
+import com.liberologico.cloudesire.common.enums.OSType;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.liberologico.cloudesire.common.enums.OSType'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AUselessImportCheck)